### DISCUSSION TITLE
Фильтрация массива альбомов

### DISCUSSION THREAD

**Discussion by svyatarom** [Chimildic/goofy#201] [Помощь с алгоритмом] [answered]
Как фильтровать массивы альбомов (релизов) по различным параметрам? Например, отсеивать альбомы с одним треком от альбомов с двумя или более треками, и наоборот.

**Comment by Chimildic** [comment, marked as answer]
Можете показать каким образом вы собираете массив альбомов? 

Можно пройтись по трекам через [rangeTracks](https://chimildic.github.io/goofy/#/reference/filter?id=rangetracks). У неё есть ключ `album_type`.
